internal/versions/versions.go

```diff
@@ -138,6 +138,9 @@ func (h *handler) V2ListSupportedOpenshiftVersions(ctx context.Context, params o
 				arch = common.DefaultCPUArchitecture
 			}
 
+			if arch == "arm64" {
+				arch = "aarch64"
+			}
```

Comment:
If we take this path, can we use both of those as `common.*`? E.g. `common.ARM64CPUArchitecture` and `common.AARCH64CPUArchitecture` or similar